tests/commands/timezone.py

```diff
@@ -154,5 +155,47 @@ def runTest(self):
         self.assert_parse_error("timezone Europe/Sofia --nontp --ntpservers=0.fedora.pool.ntp.org,1.fedora.pool.ntp.org")
 
 
+class F32_TestCase(F25_TestCase):
+    command = "timezone"
+
+    def runTest(self):
+
+        # Failures
+        # unknown argument
+        self.assert_parse_error("timezone --blah")
+        # more than two timezone specs
+        self.assert_parse_error("timezone foo bar", KickstartParseError,
+                                'One or zero arguments are expected for the timezone command')
+        self.assert_parse_error("timezone --utc foo bar", exception=KickstartParseError)
+        # no options
+        self.assert_parse_error("timezone", KickstartParseError,
+                    'At least one option and/or an argument are expected for the timezone command')
```

Comment:
When wrapping lines like this, start the next line under the function ( like you did for line 168.